vendor/engines/split_accounts/spec/models/split_account_spec.rb

```diff
@@ -0,0 +1,102 @@
+require "rails_helper"
+
+RSpec.describe SplitAccount, type: :model do
+
+  it "is an account type" do
+    expect(described_class.new).to be_an(Account)
+  end
+
+  # TODO: remove this if/when we do factory linting
+  it "has a valid factory" do
+    expect(build(:split_account)).to be_valid
+  end
+
+  describe "validations" do
+
+    context "when splits total 100 and one split has extra_penny" do
+      let(:split_account) do
+        build(:split_account, without_splits: true).tap do |split_account|
+          split_account.splits << build(:split, percent: 50, extra_penny: true, parent_split_account: split_account)
+          split_account.splits << build(:split, percent: 50, extra_penny: false, parent_split_account: split_account)
+        end
+      end
+
+      it "is valid" do
+        expect(split_account).to be_valid
+      end
+    end
+
+    context "when splits do not total 100 and one split has extra_penny" do
+      let(:split_account) do
+        build(:split_account, without_splits: true).tap do |split_account|
+          split_account.splits << build(:split, percent: 50, extra_penny: true, parent_split_account: split_account)
+          split_account.splits << build(:split, percent: 49.99, extra_penny: false, parent_split_account: split_account)
+        end
+      end
+
+      it "is invalid" do
+        expect(split_account).to_not be_valid
```

Comment:
I like your recommended approach for confirming the errors and think I'm going to use it from now on.  Do you ever take this a step further and add `expect(split_account.errors[:splits]).to include(I18n.t("activerecord.errors.models.split_account.attributes.splits.percent_total"))` ?